workshops/minecraft/readme-forge.asciidoc

```diff
@@ -107,7 +107,7 @@ Your main file is now ready to go.
 
 **Purpose:** To make the player receive items when he/she says a certain word or phrase in a chat message.
 
-**Instructions:** In your new package, `org.devoxx4kids.forge.mods`, make a new file called `ChatItems`. Replace its code with <<Chat_Items_Code>>.
+**Instructions:** In your new package, `org.devoxx4kids.forge.mods`, make a new class called `ChatItems`. To do this, right click on the package `org.devoxx4kids.forge.mods` and select `New` > `Class` then fill the name property with `ChatItems` and clico on the Finish Button. Replace its code with <<Chat_Items_Code>>. 
```

Comment:
update the type from `clico` -> `click`?

Reply:
Updated